src/main/java/org/folio/service/request/RequestBatchResourceService.java

```diff
@@ -0,0 +1,91 @@
+package org.folio.service.request;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.folio.rest.jaxrs.model.Request;
+import org.folio.service.BatchResourceService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.rits.cloning.Cloner;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+
+public class RequestBatchResourceService {
```

Comment:
This class is named liked it is a specialisation of the `BatchResourceService` yet they have a composition relationship (which I prefer).

I'd like to understand better why these two classes are split?

Reply:
`BatchResourceService` contains common methods for batch updates, but `RequestBatchResourceService` is implementation for requests batch updates and it contains it's specific logic (e.g. removing positions before assigning new).